.github/workflows/ci.yml

```diff
@@ -3,12 +3,20 @@ name: run-tests
 on: [push, pull_request]
 
 jobs:
-  test:
+  tests:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: true
       matrix:
-        laravel: [6, 8, 9]
+        php: ["8.0", "8.1"]
+        laravel: ["^9.0", "^10.0"]
+        include:
+          - laravel: "^9.0"
+            testbench: "^7.0"
+          - laravel: "^10.0"
+            testbench: "^8.0"
+        exclude:
+          - laravel: "^10.0"
+            php: "8.0"
```

Comment:
Why does `include` have `testbench` and `exclude` has `php`? Where is `testbench` used?

Reply:
Also, at this point it might be better to just use something like:
```yaml
- laravel: 9
   php: 8.0
- laravel: 10
   php: 8.1
```
That way the code will run on both PHP versions and both Laravel versions, we don't really care about L9 + 8.0 I think. And the syntax will be much simpler than having PHP versions listed, Laravel versions listed, an `include` section, and an `exclude` section.